tests/integration/helpers/common.py

```diff
@@ -369,8 +369,21 @@ def _is_workflow_run_complete(run: WorkflowRun) -> bool:
         Whether the run status is "completed".
 
     """
+    return _has_workflow_run_status(run=run, status="completed")
+
+
+def _has_workflow_run_status(run: WorkflowRun, status: str) -> bool:
```

Comment:
I introduced this because I originally wanted to access it from outside this module, but ended up not using it. I think it is still good to have this abstraction.